runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -83,6 +95,31 @@
 	@Inject
 	private ChatboxInputManager chatboxInputManager;
 
+	@Inject
+	private KeyManager keyManager;
+
+	@Inject
+	private BankTagsKeyListener keyListener;
+
+	@Setter
+	private boolean shiftHeld = false;
+
+	private boolean massEditing = false;
+
+	private List<Widget> selectedItemWidgets = new ArrayList<>();
+
+	public void startUp()
+	{
+		keyManager.registerKeyListener(keyListener);
+	}
+
+	public void shutDown()
+	{
+		keyManager.unregisterKeyListener(keyListener);
+		resetMassEdit();
+		shiftHeld = false;
```

Comment:
This still needs to reset `massEditing`.

Reply:
`resetMassEdit` does that.

Reply:
Oh right, my bad.